balancer-js/examples/pools/create/create-composable-stable-pool.ts

```diff
@@ -1,48 +1,41 @@
 /**
  * ComposableStable - Create and do an initial join.
- * Run command: yarn examples:run ./examples/pools/composable-stable/create-and-init-join.ts
+ * 
+ * Run command:
+ * yarn example ./examples/pools/create/create-composable-stable-pool.ts
  */
-import * as dotenv from 'dotenv';
-
-dotenv.config();
-import { parseFixed } from '@ethersproject/bignumber';
-
-import { ADDRESSES } from '@/test/lib/constants';
-import { setUpExample } from '../helper';
-
-import { BalancerSDK, Network, PoolType } from '@/.';
+import { BalancerSDK, Network, PoolType } from '@balancer-labs/sdk'
+import { parseFixed } from '@ethersproject/bignumber'
+import { reset, setTokenBalance, approveToken } from 'examples/helpers'
 
 async function createAndInitJoinComposableStable() {
-  const { ALCHEMY_URL: rpcUrlArchive } = process.env;
-  const network = Network.MAINNET;
-  const rpcUrlLocal = 'http://127.0.0.1:8545';
-  const sdkConfig = {
-    network,
-    rpcUrl: rpcUrlLocal,
-  };
-  const balancer = new BalancerSDK(sdkConfig);
-  const addresses = ADDRESSES[network];
-  const poolTokens = [addresses.USDC.address, addresses.USDT.address];
+  const balancer = new BalancerSDK({
+    network: Network.MAINNET,
+    rpcUrl: 'http://127.0.0.1:8545', // Using local fork for simulation
```

Comment:
Just to confirm, we're able to use ankr remote rpcURL only when we're reading state from the chain, but not writing it, right? I'm trying to understand the limitations of using one over the other.

Reply:
any example committing state changes needs a fork for simulations, read-only examples can use any rpc endpoint, including a fork, but not required.